x/authz/keys.go

```diff
@@ -0,0 +1,25 @@
+package authz
+
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "authz"
+
+	// StoreKey is the store key string for authz
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for authz
+	RouterKey = ModuleName
+
+	// QuerierRoute is the querier route for authz
+	QuerierRoute = ModuleName
+)
+
+// Keys for authz store
+// Items are stored with the following key: values
+//
+// - 0x01<granterAddressLen (1 Byte)><granterAddress_Bytes><granteeAddressLen (1 Byte)><granteeAddress_Bytes><msgType_Bytes>: Grant
+
+var (
+	// Keys for store prefixes
+	GrantKey = []byte{0x01} // prefix for each key
+)
```

Comment:
All this should imo go into keeper/keys.go

Reply:
I was thinking about it - but only the store keys (used by keeper). This is how all other modules are organizing the keys. But I agree - it's better to keep keeper keys private to the keeper. If you agree I will move the store keys to keeper.

Reply:
Yes, I was talking about the store keys only (the highlighted part in the comment). +1 for moving to keeper.